api_tests/schema_responses/views/test_schema_response_detail.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+
+from osf_tests.factories import (
+    SchemaResponseFactory,
+    RegistrationFactory,
+    AuthUserFactory
+)
+
+from osf.models import SchemaResponse
+from osf_tests.utils import DEFAULT_TEST_SCHEMA
+
+
+@pytest.mark.django_db
+class TestSchemaResponseDetail:
```

Comment:
How do you want to do these auth tests? Since it's only auth we can write really parameterized test. I would like to have these be very declarative. We should brainstorm this.